LR_Tourist/TouristWebApp/Areas/Identity/Data/TouristWebAppContext.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
```

Comment:
Why did you create `Areas` folder?
It's not consistent with whole web application)

Reply:
fixed